src/detectors/EEMC/EEMC.cc

```diff
@@ -115,40 +108,25 @@ void InitPlugin(JApplication* app) {
             "EcalEndcapNHits"
       }, // edm4hep::SimCalorimeterHitCollection
 #endif
-      {"EcalEndcapNTruthClustersWithoutShapes",             // edm4eic::Cluster
-       "EcalEndcapNTruthClusterAssociationsWithoutShapes"}, // edm4eic::MCRecoClusterParticleAssociation
+      {"EcalEndcapNTruthClusters",             // edm4eic::Cluster
+       "EcalEndcapNTruthClusterAssociations"}, // edm4eic::MCRecoClusterParticleAssociation
       {.energyWeight = "log", .sampFrac = 1.0, .logWeightBase = 4.6, .enableEtaBounds = false},
       app // TODO: Remove me once fixed
       ));
 
-  app->Add(new JOmniFactoryGeneratorT<CalorimeterClusterShape_factory>(
-      "EcalEndcapNTruthClusters",
-      {"EcalEndcapNTruthClustersWithoutShapes", "EcalEndcapNTruthClusterAssociationsWithoutShapes"},
-      {"EcalEndcapNTruthClusters", "EcalEndcapNTruthClusterAssociations"},
-      {.energyWeight = "log", .logWeightBase = 4.6}, app));
-
   app->Add(new JOmniFactoryGeneratorT<CalorimeterClusterRecoCoG_factory>(
-#if EDM4EIC_VERSION_MAJOR >= 8
-      "EcalEndcapNClustersWithoutPIDAndShapes",
-#else
-      "EcalEndcapNClustersWithoutShapes",
-#endif
+      "EcalEndcapNClusters",
       {
```

Comment:
We really have to figure out why (and to what extent) these reverts have happened.